.github/labeler.yml

```diff
@@ -0,0 +1,63 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Could you add copyright info when you add header?  and you can also add header for other `yml` files